src/services/sync/strategies/TSP/TSP.js

```diff
@@ -0,0 +1,164 @@
+const rp = require('request-promise-native');
+const url = require('url');
+const moment = require('moment');
+const { JWE, JWK, JWS } = require('jose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'jose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)